exercises/nth-prime/example.js

```diff
@@ -1,36 +1,28 @@
-let realPrimes = [];
-
-function generatePrimes(uptoNumber) {
-  if (realPrimes.length > 0) {
-    return realPrimes;
-  }
-
-  let currentPrime,
-    possiblePrimes = [];
-  for (let i = 2; i <= uptoNumber; i++) {
-    possiblePrimes.push({ number: i, prime: true });
-  }
-
-  for (let i = 2; i < Math.sqrt(possiblePrimes.length); i++) {
-    for (let j = 0; j < possiblePrimes.length; j++) {
-      currentPrime = possiblePrimes[j];
-      if (currentPrime.number !== i && currentPrime.number % i === 0) {
-        currentPrime.prime = false;
+const isPrime = number => {
+  if (number === 2 || number === 3) { return true; }
+  if (number % 2 === 0) { return false; }
+  for (let i = 3; i <= Math.sqrt(number); i += 2) {
+      if (number % i === 0) {
+          return false;
       }
-    }
   }
-
-  return possiblePrimes
-    .filter(candidate => candidate.prime)
-    .map(p => p.number);
+  return true;
 }
 
-export class Prime {
-  nth(nthPrime) {
-    if (nthPrime === 0) {
-      throw new Error('Prime is not possible');
-    }
-    realPrimes = generatePrimes(200000);
-    return realPrimes[nthPrime - 1];
+export const prime = ordinal => {
+  if (ordinal === 0) {
+      throw new Error('there is no zeroth prime');
+  }
+  if (ordinal === 1) {
+      return 2;
```

Comment:
Can you change the indentation to 2 spaces for consistency?

Reply:
Done.